src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/XslTransformMultith.cs

```diff
@@ -29,7 +29,7 @@ public SameInstanceXslTransformTestCase(ITestOutputHelper output) : base(output)
         public new void Init(object objParam)
         {
             xsltSameInstance = new XslCompiledTransform();
-            _strPath = Path.Combine(@"TestFiles\", FilePathUtil.GetTestDataPath(), @"XsltApiV2\");
```

Comment:
@sepidehMS The changes in this PR do not depend on the other PR. The change in the XmlResolver to no longer prepend the file// scheme cause backslashes to be maintained. These backslashes lead to invalid paths on Unix.